spec/index.bs

```diff
@@ -15004,7 +15004,7 @@ The clipped shader output |c|.v is produced based on the interpolation qualifier
         The interpolation ratio gets adjusted against the perspective coordinates of the
```

Comment:
related: the description of flat interpolation above refers to "first vertex" which is outdated, it depends on whether it's `first` or `either`: https://gpuweb.github.io/gpuweb/wgsl/#interpolation

Reply:
Updated.